api/timeline/v1/timeseries.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package clutch.timeline.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/timeline/v1;timelinev1";
+
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+
+// A timeseries datapoint message is useful for organizing events
+// to be displayed in a timeline view.  Users can transform data
+// into a timeseries format and be able to organize them.
+message TimeseriesData {
+  string project_name = 1 [ (validate.rules).string = {min_bytes : 1} ];
```

Comment:
should we call this project name? It seems like time series data could be used for things even outside of projects.